unity/Assets/Scripts/QuestEditor/EditorSelectionList.cs

```diff
@@ -152,6 +154,11 @@ public void SelectItem(UnityEngine.Events.UnityAction call)
                 tb = new TextButton(new Vector2(21, offset), new Vector2(20, 1), 
                     new StringKey(null, filtered[i].name, false), delegate { SelectComponent(key); }, filtered[i].color);
                 tb.button.GetComponent<UnityEngine.UI.Text>().fontSize = UIScaler.GetSmallFont();
+                if (key == null || key.Length == 0)
```

Comment:
Yes, but they aren't currently. I'd prefer to replace with something like {none} or {empty}. I have added the check into the delete function anyway for this particular bug.